src/SumoCoders/FrameworkCoreBundle/BreadCrumb/BreadCrumbBuilder.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\BreadCrumb;
+
+use Knp\Menu\FactoryInterface;
+use Knp\Menu\MenuItem;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\HttpFoundation\Request;
+use SumoCoders\FrameworkCoreBundle\Event\ConfigureMenuEvent;
+
+final class BreadCrumbBuilder
+{
+    /**
+     * @var bool
+     */
+    private $dontExtractFromTheRoute = false;
```

Comment:
Het zou beter zijn om hier "extractFromTheRoute" van te maken, en default op true te zetten, dat is een properdere variabele naam.